packages/fxa-settings/src/components/LinkExpired/index.tsx

```diff
@@ -8,21 +8,12 @@ import { LinkType } from '../../lib/types';
 import CardHeader from '../CardHeader';
 
 type LinkExpiredProps = {
-  linkType: LinkType;
+  linkType: LinkType,
```

Comment:
Interesting. Do you by chance have a feeling for what is most prevalent?

Reply:
My guts (which probably aren't a metric we should really measure against) says `;` is the most common.
I'd also bet that there is an ESLint rule for that, and I'd have to poke at it and see if Prettier has a preference.

But here is a mediocre <kbd>grep</kbd> invocation that you can tweak to your liking:

```sh
git grep -En -A3 "^(export )?type [A-Za-z]{2,} = \{" | grep -Ev "\d\-$" | head -30

packages/functional-tests/lib/fixtures/standard.ts:16:export type TestOptions = {
packages/functional-tests/lib/fixtures/standard.ts-17-  pages: POMS;
packages/functional-tests/lib/fixtures/standard.ts-18-  credentials: Credentials;
packages/functional-tests/lib/fixtures/standard.ts-19-};
packages/functional-tests/lib/fixtures/standard.ts:20:export type WorkerOptions = { targetName: TargetName; target: ServerTarget };
--
packages/fxa-admin-panel/src/components/AppLayout/index.tsx:13:type AppLayoutProps = {
packages/fxa-admin-panel/src/components/AppLayout/index.tsx-14-  children: any;
packages/fxa-admin-panel/src/components/AppLayout/index.tsx-15-};
--
packages/fxa-admin-panel/src/components/Guard/index.tsx:10:export type GuardProps = {
packages/fxa-admin-panel/src/components/Guard/index.tsx-11-  features: AdminPanelFeature[];
packages/fxa-admin-panel/src/components/Guard/index.tsx-12-};
--
packages/fxa-admin-panel/src/components/PageAccountSearch/DangerZone/index.tsx:13:type DangerZoneProps = {
packages/fxa-admin-panel/src/components/PageAccountSearch/DangerZone/index.tsx-14-  uid: string;
packages/fxa-admin-panel/src/components/PageAccountSearch/DangerZone/index.tsx-15-  email: Email;
packages/fxa-admin-panel/src/components/PageAccountSearch/DangerZone/index.tsx-16-  disabledAt: number | null;
--
packages/fxa-admin-server/scripts/audit-tokens.ts:25:type TargetTable = { name: string; keyCol: string };
--
packages/fxa-auth-client/lib/client.ts:28:export type SignUpOptions = {
packages/fxa-auth-client/lib/client.ts-29-  keys?: boolean;
packages/fxa-auth-client/lib/client.ts-30-  service?: string;
packages/fxa-auth-client/lib/client.ts-31-  redirectTo?: string;
--
...
```

Per https://medium.com/swlh/typescript-best-practices-delimiters-and-ordering-7feb42e0ca1d:
> _"They’re both valid, but TypeScript prefers semicolon."_

Still trying to Google to find which ESLint rule might enforce this so I can scan the repo locally after tweaking a few .eslintrc.js config files.

Reply:
Behold! https://typescript-eslint.io/rules/member-delimiter-style

But this colorful warning has me intrigued:
> _**DANGER**_
> _We strongly recommend you do not use this rule or any other formatting linter rules. Use a separate dedicated formatter instead. See [What About Formatting?](https://typescript-eslint.io/linting/troubleshooting/formatting) for more information._

But as for our internal style guide&hellip; looking at the https://mozilla.github.io/ecosystem-platform/reference/style-guides/node-style-guide#style docs says:

> _"We use the [Google TypeScript Style Guide](https://google.github.io/styleguide/tsguide.html#source-organization) for our base code style."_

&hellip; and I think the relevant docs are https://google.github.io/styleguide/tsguide.html#member-property-declarations, which claims:
> _"Interface and class declarations must use a semicolon to separate individual member declarations:"_

I mean, this really is the tiniest of nits that can just as easily be ignored (the easiest of all solutions). Just following see-something-file-something rules.

Reply:
Sounds good. I'll just switch it semis and add new lines. Seems the most basic way 👍 . Thanks for the all the digging!

Reply:
Had a couple seconds to poke at this today and looks like Prettier prefers semicolons:

```sh
npx prettier --config _dev/.prettierrc packages/fxa-settings/src/components/LinkExpired/index.tsx
```

I wonder if this means the Husky precommit hooks aren't working 100% of the time, since I thought this should match:
https://github.com/mozilla/fxa/blob/27b8af1032c406d47b5b5e2028335567604b85bf/package.json#L61-L63

Reply:
Interesting. I had format on save off in my editor, which might explain some things. I'll keep an a closer eye on husky regardless. It does seem like something is off there.